d2renderers/d2svg/d2svg.go

```diff
@@ -224,6 +227,39 @@ func arrowheadMarker(isTarget bool, id string, connection d2target.Connection) s
 				width*0.6, height*7/8,
 			)
 		}
+	case d2target.FilledCircleArrowhead:
+		attrs := fmt.Sprintf(`class="connection" fill="%s" stroke-width="%d"`, connection.Stroke, connection.StrokeWidth)
+		offset := 4.0 + float64(connection.StrokeWidth*2)
+		if isTarget {
+			path = fmt.Sprintf(`<circle %s cx="%f" cy="%f" r="%f"/>`,
+				attrs,
+				offset+9, height/2,
```

Comment:
why does offset changes for target/source?

Reply:
That I am not sure of. It would render 3-5 pixels off on opposite sides of the connection. The AB and CD blocks had a reversed arrow direction it seemed. Will explore in a few.

Reply:
Removed and fixed